src/rustdoc-json-types/lib.rs

```diff
@@ -281,19 +281,28 @@ pub enum StructType {
     Unit,
 }
 
+#[non_exhaustive]
+#[serde(rename_all = "snake_case")]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub enum Modifiers {
+    Const,
+    Unsafe,
+    Async,
```

Comment:
Should this make the ABI part of the `Modifiers`? I guess that's hard to represent in JSON because it has a value; maybe keeping it separate is better.

Reply:
I think keeping it separate is better for the JSON. The most palatable option that comes to mind is making a "fallback" `Modifier::Abi(String)`, and that only works if an ABI and modifier name can never clash.